stories/topics/proc-azure-configure-ad-sso.adoc

```diff
@@ -75,6 +75,59 @@ Add the key _(Application (client) ID)_ and value (_Value_) of the secret that y
   . In *{MSEntraID} OAuth2 Secret*, paste the secret _Value_.
 . Click *Save*.
 
+.Creating a service principal for automating Azure resources
+
+[role="_abstract"]
+To enable the {PlatformNameShort} application to access and manage Azure resources, you must provide authorization credentials after deployment.
+The Microsoft Azure collection supports service principal authentication.
+
+To create a service principal, you must have administrator privileges with tenancy-wide permissions on your Azure tenant.
+Your {AAPonAzureNameShort} deployment provisions in the same Subscription ID as the service principal created in this step.
+
+. Navigate to the Azure portal
+. Click the link:https://docs.microsoft.com/en-us/azure/cloud-shell/overview[Cloud Shell] icon to open a bash Cloud Shell in your browser.
+. Set the Azure CLI to use the subscription that you intend to use for automating Azure services. Run the following command from the shell:
++
+-----
+az account set --subscription <your_subscription_id>
+-----
+. Run the following command using the Azure CLI to create a privileged service principal in {MSEntraID}:
++
+-----
+az ad sp create-for-rbac --name ansible --role Contributor
+-----
++
+The output displays the _appID_ and _tenant_ keys for the service principal:
++
+-----
+{
+  "appId": "xxxxxxx-xxx-xxxx",
+  "displayName": "ansible",
+  "name": "xxxxxxx-xxx-xxxx",
+  "password": "xxxxxxx-xxx-xxxx",
+  "tenant": "xxxxxxx-xxx-xxxx"
+}
+-----
+. Store the service principal details securely, as they are displayed only when you create the secret. You need them when you deploy Automation controller.
+See link:{LinkControllerAdminGuide} for help with deploying your automation controller.
+
+
+.Maintaining your service principals
+
+Service principal credentials have a limited lifetime that is set in your {MSEntraID} configuration.
+Track the lifespan of the service principal if you intend to automate against Azure for an extended period of time.
+You can create a new one when needed.
+
+To view records of updated or deleted service principles, run the following Azure CLI command:
```

Comment:
This terminology is new to me but "service principles" here is different from the other instances where it's "service principals" so just flagging incase 👍